lib/tilt/commonmarker.rb

```diff
@@ -16,8 +20,9 @@ class CommonMarkerTemplate < Template
       :GITHUB_PRE_LANG,
       :HARDBREAKS,
       :NOBREAKS,
-      :SAFE,
```

Comment:
According to the current source code, SAFE was removed in 0.18.0.  Does removing this break compatibility with CommonMarker <0.18.0?